vector-im/element-android
# Missing translations on "replyTo" messages (PSG-978)

## Type of change


## Content


Adding translations for "replyTo" events message on application side.
The description parameter exposed in SDK API for the start of a live location has also been removed to avoid introducing translated text in Matrix events.

## Motivation and context


Closes #7555 

## Screenshots / GIFs





<img src="https://user-images.githubusercontent.com/46314705/201154879-52b7c71f-38f1-4cff-b74a-f1e070da3632.png" width=300 /> <img src="https://user-images.githubusercontent.com/46314705/201154897-700e3b31-8d06-4edc-9cdd-cf11a8c10f33.png" width=300 />
<img src="https://user-images.githubusercontent.com/46314705/201155018-5440b012-85ce-4b1f-9326-a5fc779c19b3.png" width=300 /> <img src="https://user-images.githubusercontent.com/46314705/201155037-5160c999-f68d-47c9-9144-df5d0f302565.png" width=300 />

## Tests



- Go to a room (encrypted or not)
- Send different type of messages
- Reply to these messages
- Check the replies are well rendered (right now they are not translated: we should wait for Weblate sync to see the changes)

## Tested devices

- OS version(s): Android 11

## Checklist
